src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
+    }
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT + 1);
+        switch (figureNumber) {
+            case 1:
+                return createRandomSquare();
+            case 2:
+                return createRandomRectangle();
+            case 3:
+                return createRandomRightTriangle();
+            case 4:
+                return createRandomCircle();
+            case 5:
+                return createRandomIsoscelesTrapezoid();
+            default:
+                return getDefaultFigure();
+        }
```

Comment:
1. There is no need to add 1, we may just simply start checking from zero
2. We shouldn't return default figure from getRandomFigure() method. If we negatively checked first 4 options then it's obviously last one
```suggestion
        int figureNumber = random.nextInt(FIGURE_COUNT);
        switch (figureNumber) {
            case 0:
                return createRandomSquare();
            case 1:
                return createRandomRectangle();
            case 2:
                return createRandomRightTriangle();
            case 3:
                return createRandomCircle();
            default:
                return createRandomIsoscelesTrapezoid();
        }
```